components/support/ktx/src/main/java/mozilla/components/support/ktx/android/content/Context.kt

```diff
@@ -33,3 +35,11 @@ fun Context.isOSOnLowMemory(): Boolean {
         activityManager.getMemoryInfo(memoryInfo)
     }.lowMemory
 }
+
+/**
+ * Returns if a list of permission have been granted, if all the permission have been granted
+ * returns true otherwise false.
+ */
+fun Context.isPermissionGranted(vararg permission: String): Boolean {
+    return permission.all { checkSelfPermission(this, it) == PERMISSION_GRANTED }
+}
```

Comment:
Let's make sure to add this (and of course the new component!) to the changelog. :)

Reply:
Done!
https://github.com/mozilla-mobile/android-components/pull/1032